### ISSUE TITLE
(Enhancement) Sound activation for Captcha?

### ISSUE THREAD

**Issue by ifhbiff** [PokeAlert/PokeAlert#291]
Why can I get a sound for when the scan finds a Pokemon, but not for (the much more common nowadays) captcha alert?  I have to constantly be looking at my screen because the messages pile up and unless I'm watching when the alert comes up, I frequently get expired Captchas ... which I assume is not healthy for the survival of my accounts.

Perhaps a sound option for Pokemon notification, Captcha notification, or both?

**Comment by kittymjau** [comment]
Yes, its annoying that Pokemon and Captcha has the same sound. Needs to be changed.

**Comment by ifhbiff** [comment]
Uhh that's not what I said.  Is there a sound for captcha warnings now, because there wasn't when I wrote this.
